tests/MoBi.Tests/Presentation/EditTableFormulaWithOffSetFormulaPresenterSpecs.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using FakeItEasy;
+using MoBi.Core.Domain.Model;
+using MoBi.Core.Domain.Services;
+using MoBi.Helpers;
+using MoBi.Presentation.DTO;
+using MoBi.Presentation.Mappers;
+using MoBi.Presentation.Presenter;
+using MoBi.Presentation.Views;
+using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Commands.Core;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Formulas;
+using OSPSuite.Core.Domain.UnitSystem;
+using OSPSuite.Core.Services;
+using OSPSuite.Presentation.Core;
+
+namespace MoBi.Presentation
+{
+   public abstract class concern_for_EditTableFormulaWithOffSetFormulaPresenter : ContextSpecification<IEditTableFormulaWithOffSetFormulaPresenter>
```

Comment:
@Yuri05 Could you review that PR: Especially, where should we allow TableFormulaWithOffset and SumFormula ?